scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+SCRIPT_DIR="$(realpath $(dirname $0))"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+OUTPUT="$3"
+
+echo -n "\"use strict\";" > "${OUTPUT}"
+echo -n "var backup = { module: module, exports: exports, Browser: Browser, backup: backup, brotli: brotli };" >> "${OUTPUT}"
+echo -n "var Browser = {T:function(){}};var exports = {};var module = { exports: exports };" >> "${OUTPUT}"
```

Comment:
Depending on which compression algorithm we'd choose and how easy the decompression counterpart let's itself be encapsulated, we could probably avoid crap like this :-).